db/backup_download.go

```diff
@@ -50,8 +53,12 @@ func DownloadBackup(app, addon, backupID string, opts DownloadBackupOpts) error
 		if len(backups) == 0 {
 			return errgo.New("This addon has no backup")
 		}
-		backupID = backups[0].ID
+		for _, backup := range backups {
+			backupIDs = append(backupIDs, backup.ID)
+		}
```

Comment:
I wouldn't do it that way. I would call here a method, for instance:

```go
backup, err := getLastSuccessfulBackup(ctx, backups)
```

And that's it for the modification in this method. 

Here, you modified the behaviour of the method when a customer calls `backup-download` with a backup ID in argument. But we don't want to modify this behaviour.